### PR TITLE
Dev: Static Analysis: add clang tools to Ubuntu install instructions

### PR DESCRIPTION
**Pull request by IamPete1**
I also had to install clang tools, set both to version 7 to match CI

### CODE REVIEW

File: dev/source/docs/static-analysis.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ You can do your own static analysis using autotest:
 
    You may need to install clang for this to work.
 
-   On Ubuntu, "sudo apt-get install clang"
+   On Ubuntu, `sudo apt-get install clang-7`, `sudo apt-get install clang-tools-7`
```
Comment (khancyr) [inline dev/source/docs/static-analysis.rst:23]: Why not put them on the same cmd ?

**Comment by IamPete1** [comment]
Turns out using edit in github I made a new branch in the AP repo, not mine as it used to. Oops, we should delete this branch once it gets merged.

File: dev/source/docs/static-analysis.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ You can do your own static analysis using autotest:
 
    You may need to install clang for this to work.
 
-   On Ubuntu, "sudo apt-get install clang"
+   On Ubuntu, `sudo apt-get install clang-7 clang-tools-7`
```
Comment (peterbarker) [inline dev/source/docs/static-analysis.rst:23]: Is this going to be wrong on older versions of Ubuntu?

Comment (IamPete1) [reply dev/source/docs/static-analysis.rst:23]: I have no idea, i'm on 18.04 and it worked for me. We could just mention the need for it and not provide the command.....

Comment (khancyr) [reply dev/source/docs/static-analysis.rst:23]: yep, doesn't work on 20.04 ...
so `sudo apt-get install clang clang-tools` will get you the latest available on ubuntu

Comment (IamPete1) [reply dev/source/docs/static-analysis.rst]: removed the version numbers, should work for everyone ?

**Review by peterbarker — APPROVED** [review]

**Comment by Hwurzburg** [comment]
merging

**Pull request merged**